src/bin/sus-kernel/main.rs

```diff
@@ -11,11 +11,12 @@ mod log;
 mod permission;
 mod request;
 
-use permission::verify::AbstractVerifier;
-use request::Request;
-
+use permission::verify::from_sudoers;
+// use permission::verify::AbstractVerifier;
+use crate::permission::verify::Verifier;
 #[cfg(feature = "log")]
 use log::AbstractLogger;
+use request::Request;
```

Comment:
Maybe move this `use` into a different block separated by whitespace. That way, we can keep them grouped logically by function.